lib/rust/ensogl/core/src/system/js/app.rs

```diff
@@ -60,8 +66,16 @@ pub mod js_bindings {
     mock_data! { Param => JsValue }
 
     impl App {
-        pub fn register_get_shaders_rust_fn(&self, _closure: &Closure<dyn FnMut() -> JsValue>) {}
-        pub fn register_set_shaders_rust_fn(&self, _closure: &Closure<dyn FnMut(JsValue)>) {}
+        pub fn register_get_dynamic_assets_sources_rust_fn(
+            &self,
+            _closure: &Closure<dyn FnMut() -> JsValue>,
+        ) {
+        }
+        pub fn register_set_dynamic_asset_rust_fn(
+            &self,
+            _closure: &Closure<dyn FnMut(JsValue, JsValue, JsValue)>,
+        ) {
+        }
```

Comment:
why the formatting changed here? The one-liners looked much better. EDIT: Ah, the name is longer. Hmm, can we maybe rename the argument to make it shorter and still preserve the fn in one line?

Reply:
Renaming `_closure` doesn't shorten it enough, it just has such a long name.